client/components/payment-activity/payment-activity-data.tsx

```diff
@@ -43,6 +43,30 @@ const PaymentActivityData: React.FC = () => {
 	const refunds = paymentActivityData?.refunds ?? 0;
 	const currency = paymentActivityData?.currency;
 
+	const searchTermsForViewReportLink = {
+		totalPaymentVolume: [
+			'charge',
+			'payment',
+			'payment_failure_refund',
+			'payment_refund',
+			'refund',
+			'refund_failure',
+			'dispute',
+			'dispute_reversal',
+			'card_reader_fee',
+		],
+	};
+
+	const getSearchParams = ( searchTerms: string[] ) => {
```

Comment:
💭 This is not an issue per se, but I'd consider moving this function and `searchTerms` declaration outside of the React component scope (alongside `getDateRange`) since it doesn't require any component variables/state.

E.g. if it is declared outside the component, it explicitly lets us know it is decoupled from the component and can be used elsewhere, moved, etc.

Reply:
That is a very valuable feedback. Note-to-self on deciding where to scope. 🙌 .

Reply:
done here! 4b53bac